noetic/distribution.yaml

```diff
@@ -12242,8 +12242,8 @@ repositories:
     release:
       tags:
         release: release/noetic/{package}/{version}
-      url: https://github.com/flynneva/udp_msgs-release.git
-      version: 0.0.4-1
+      url: git@github.com:ros2-gbp/udp_msgs-release.git
```

Comment:
Please keep the https URL. SSH URLs require authentication with github to download.

Reply:
@sloretz back to you, sorry about that 👍🏼